### DISCUSSION TITLE
Is there a webhook for when a coupon is used?

### DISCUSSION THREAD

**Discussion by erwinrommel** [swellstores/community#500] [Help] [answered]
We are using 100% discounted orders with coupons and wanted to handle events when a coupon is used.

Is there a webhook that we can receive when a coupon code is applied to an order?

e.g. `coupon.code.updated`

**Comment by ericingram** [comment, marked as answer]
There is no standard event specifically for this, though it could be done with apps.

**Reply by logeshswell** [reply]
**Alternate workaround:**
Can you listen for `order.created` or/and `order.updated` events and check whether your expected coupon code is applied?

**Reply by erwinrommel** [reply]
Thank you. We will go with this work around for now since we are already listening to order updated events.
